opentelemetry-api/src/opentelemetry/context/propagation/httptextformat.py

```diff
@@ -93,9 +99,120 @@ def extract(
 
         """
 
+
+class HTTPInjector(abc.ABC):
+    @classmethod
```

Comment:
Why make these classmethods? Is it an error to ever instantiate an injector/extractor?